src/App.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-
+import AuthorizationButton from './components/AutorizationButton/AuthorizationButton';
```

Comment:
don't over nest, take Authorization button out of the directory. Put it directly inside components

Reply:
I don't know Osama said that would be better for them to be in inside a folder